### PR TITLE
Improve writing

### PR DESCRIPTION
**Pull request by jorgebucaran**
Some improvements to the documentation. Let me know if there's anything else you'd like to me to see.

Frankly, I didn't understand very well the information about variables in [Configuration](https://github.com/acomagu/fish-async-prompt#configuration). I also suggest rewriting that part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,24 +16,26 @@ With [Fisher](https://github.com/jorgebucaran/fisher):
 $ fisher install acomagu/fish-async-prompt
 ```
 
-If your prompt doesn't work correctly, try changing the configuration.
```
Comment (acomagu) [inline README.md:19]: Why is it not necessary? I mean, I want the user to tune the configurations if they have problems.

**Comment by acomagu** [comment]
@jorgebucaran Thanks! Actually I'm not good at English so your suggestion is really helpful for me and this repository.

Could you tell me what is unclear for you in the configuration part? Because I'm not sure about how can I improve that.

**Comment by acomagu** [comment]
How's that?

**Comment by jorgebucaran** [comment]
That looks @acomagu! Let's go with that. 💯

**Comment by acomagu** [comment]
Thanks a lot!!

**Pull request merged**